worldedit-core/src/main/resources/lang/strings.json

```diff
@@ -301,6 +303,58 @@
+    "worldedit.error.max-radius": "Maximum radius (in configuration): {0}",
+    "worldedit.error.unknown-direction": "Unknown direction: {0}",
+    "worldedit.error.empty-clipboard": "Your clipboard is empty. Use //copy first.",
+    "worldedit.error.invalid-filename": "Filename '{0}' invalid: {1}",
+    "worldedit.error.invalid-filename.invalid-characters": "Invalid characters or extension missing",
+    "worldedit.error.file-resolution": "File '{0}' resolution error: {1}",
+    "worldedit.error.file-resolution.outside-root": "Path is outside allowable root",
+    "worldedit.error.file-resolution.resolve-failed": "Failed to resolve path",
+    "worldedit.error.file-aborted": "File selection aborted.",
+    "worldedit.error.no-file-selected": "No file selected.",
+    "worldedit.error.world-unloaded": "The world was unloaded already.",
+    "worldedit.error.not-a-block": "This item is not a block.",
+    "worldedit.error.not-a-block.item": "The item '{0}' is not a block.",
+    "worldedit.error.incorrect-usage": "Usage: {0}",
+    "worldedit.error.invalid-page": "Invalid page number",
+    "worldedit.error.parser.bad-state-format": "Bad state format in {0}",
+    "worldedit.error.parser.duplicate-property": "Duplicate property: {0}",
+    "worldedit.error.parser.empty-state": "Empty part in state",
+    "worldedit.error.parser.missing-equals-separator": "Missing '=' separator",
+    "worldedit.error.parser.clipboard.missing-offset": "Offset specified with @ but no offset given. Use '#copy@[x,y,z]'.",
+    "worldedit.error.parser.unknown-property": "Unknown property '{0}' for block '{1}'",
+    "worldedit.error.parser.unknown-value": "Unknown value '{0}' for property '{1}'",
+    "worldedit.error.parser.empty-property": "Empty property in state",
+    "worldedit.error.parser.empty-value": "Empty value in state",
+    "worldedit.error.parser.invalid-colon": "Invalid colon.",
+    "worldedit.error.parser.invalid-expression": "Invalid expression: {0}",
+    "worldedit.error.parser.negate-nothing": "Cannot negate nothing!",
+    "worldedit.error.parser.hanging-lbracket": "Invalid format. Hanging bracket @ '{0}'.",
+    "worldedit.error.parser.missing-rbracket": "State is missing trailing ']'",
+    "worldedit.error.parser.missing-random-type": "Missing the type after the % symbol for '{0}'",
+    "worldedit.error.parser.clipboard.missing-coordinates": "Clipboard offset needs x,y,z coordinates.",
+    "worldedit.error.disabled": "This functionality is disabled (see WorldEdit configuration).",
+    "worldedit.error.unknown": "Unknown error occurred: {0}",
+    "worldedit.error.player-only": "This user is not a player!",
+    "worldedit.error.missing-extent": "No Extent is known",
+    "worldedit.error.missing-session": "No LocalSession is known",
+    "worldedit.error.missing-world": "You need to provide a world (Try //world)",
+    "worldedit.error.missing-actor": "No world is known (try //world)",
```

Comment:
Capitalization inconsistent here with `try //world` vs `Try //world` above.

Reply:
The string in general is just wrong :P